modules/hooks.nix

```
@@ -359,6 +359,13 @@ in
           entry = "${pkgs.python3Packages.black}/bin/black";
           types = [ "file" "python" ];
         };
+      ruff =
+        {
+          name = "ruff";
+          description = " An extremely fast Python linter, written in Rust.";
+          entry = "${pkgs.python3Packages.black}/bin/ruff";
+          types = [ "file" "python" ];
```

```suggestion
          types = [ "python" ];
```

The type `file` is not necessary when using the `python` type.